### PR TITLE
feat: Expand pl_PL translations to latest translation spec

### PR DESCRIPTION
**Pull request by MichalPodeszwa**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
N/a

### 💡 Background and solution
I've noticed that there aren't polish translations for form validation. When adding this I figured I'll go ahead and add all missing translations (basing them on `default.tsx`).

I haven't seen a place in code where I could see all translations (I've tested this using various components and wrapping them in docs with ConfigProvider). Is there one place where we can see all translations?

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Added missing pl_PL translations |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/locale/pl_PL.tsx [hunk 1]
```diff
@@ -44,6 +65,71 @@ const localeValues: Locale = {
...
+    expand: 'Rozwiń',
+  },
+  PageHeader: {
+    back: 'Wstecz',
+  },
+  Form: {
+    optional: '(opcjonalne)',
+    defaultValidateMessages: {
+      default: 'Błąd walidacji dla pola ${label}',
+      required: 'Wprowadź ${label}',
```
Comment (Piotr-Musial) [inline components/locale/pl_PL.tsx:84]: Przykładowy label = "Data urodzenia".

Może zamiast "Wprowadź Data urodzania", bardziej miałoby sens "Pole Data urodzenia jest wymagane", albo "Wprowadź wartość dla pola: Data urodzenia"?

File: components/locale/pl_PL.tsx [hunk 2]
```diff
@@ -44,6 +65,71 @@ const localeValues: Locale = {
...
+      enum: 'Pole ${label} musi posiadać wartość z listy: [${enum}]',
+      whitespace: '${label} Nie może być pustym znakiem',
```
Comment (Piotr-Musial) [inline components/locale/pl_PL.tsx:86]: Może tak jak wyżej, czyli "Pole ${label} nie może być puste"?

File: components/locale/pl_PL.tsx [hunk 3]
```diff
@@ -44,6 +65,71 @@ const localeValues: Locale = {
...
+      },
+      string: {
+        len: '${label} musi posiadać ${len} znaków',
+        min: '${label} musi posiadać co namniej ${min} znaków',
+        max: '${label} musi posiadać maksymalnie ${max} znaków',
+        range: '${label} musi posiadać między ${min} a ${max} znaków',
+      },
+      number: {
+        len: '${label} musi mieć wartość o długości ${len}',
+        min: '${label} musi być większe lub równe ${min}',
```
Comment (Piotr-Musial) [inline components/locale/pl_PL.tsx:115]: Przykład: "Wiek musi być większe lub równe 18"
Może lepiej tak jak wyżej, czyli: "${label} musi mieć wartość większą lub równą..."?

File: components/locale/pl_PL.tsx [hunk 4]
```diff
@@ -44,6 +65,71 @@ const localeValues: Locale = {
...
+        max: '${label} musi być mniejsze lub równe ${max}',
```
Comment (Piotr-Musial) [inline components/locale/pl_PL.tsx:116]: Analogicznie jak w komentarzu przy `min`.

File: components/locale/pl_PL.tsx [hunk 5]
```diff
@@ -44,6 +65,71 @@ const localeValues: Locale = {
...
+        range: '${label} musi być pomiędzy ${min} a ${max}',
+      },
+      array: {
+        len: '${label} musi posiadać ${len} elementów',
+        min: '${label} musi posiadać co najmniej ${len} elementów',
+        max: '${label} musi posiadać maksymalnie ${len} elementów',
+        range: '${label} musi posiadać między ${min} a ${max} elementów',
+      },
+      pattern: {
+        mismatch: '${label} nie posiada wartości zgodnej z wzorem ${pattern}',
```
Comment (Piotr-Musial) [inline components/locale/pl_PL.tsx:126]: "z wzorem" -> "ze wzorem"?

**Review by Piotr-Musial — APPROVED** [review]

**Comment by MichalPodeszwa** [comment]
Thank you @Piotr-Musial - great suggestions!

**Review by yoyo837 — APPROVED** [review]

**Pull request merged**